index.html

```diff
@@ -0,0 +1,127 @@
+            every step of the way. Manny simply outboxed him. But then when the decision came, Bradley was awarded the victory to the shock
+            of many different individuals. The second match went exactly like the first with Manny Pacquiao's skills on full display.
+             <div class="container"><p class="container">He was
+            punishing Bradley at every turn and this time the judges scored accordingly. Manny came to leave no doubt that he had won the
+            fight. It was as impressive of a performance as we had seen in his career and served as redemption for the first fight that these
+            two had. Which takes us to the most recent bout on April 9th,2016. It was a fight for the ages with each round becoming more
+            impressive than the last. Both fighters were landing more often than not in quick bursts of action. Pacquiao was throwing a
+            series of quick combinations and landed a short hook that put Bradley down on the canvas. The knockdown proved to be nothing
+            more than a slip in the eyes of many. Then again in the 9th round Manny lands a huge shot on Bradley knocking him down for the
+            second time in the fight to. There was no doubt about this lethal blow as Bradley did a backwards somersault and almost careened
+            into the stands. After that Bradley was not the same and Manny was able to close out the fight in convincing fashion earning him
+            his 58th career professional boxing victory.</p></div><br> Afterwards in the ring Max Kellerman asked Manny if the rumors were true
+            and if he was in fact retiring to which Manny replied "Yes". He informed us of the bigger issues he wished to face outside of
+            Like the state of his home country The Philippines which is filled with corruption, crime, poverty as we well as many other
+            societal discrepancies. He is after all, a politician back home and uses his wealth and public status to assist the nation in which
+            he came from. We know another big name fighter by the name of Floyd "Money" Mayweather, who Manny fought and lost to last year,
+            also announced his retirement back in September, so the question on everyones mind is whether or not Manny is retired for just
+            the time being, or if he is saving himself for a mega-rematch, between him and Floyd? Nonetheless if this really is it, It has
+            been a pleasure watching one of the greatest fighters to ever live perform in the boxing ring for the better part of a decade.</p>
+          <section class="blog-body"></section>
+          <a href="#" class="read-on">Read on &rarr;</a>
+        </article>
+      </section>
+      <footer class="footer">
+          &copy; 2016 | Ruben Sarkisyan | Sports Blog
+      </footer>
+  </body>
+  <script src="scripts/blog.js"
```

Comment:
You need to close your script tag, otherwise it won't work. <script src=""></script>